Sign Helm charts (#374)
hivemq/helm-charts: README.md

## Tests
Integration tests are split into two Gradle submodules. One submodule for the HiveMQ Platform Operator (new): [`tests-hivemq-platform-operator`](./tests-hivemq-platform-operator) and another submodule for the HiveMQ Operator (legacy): [`tests-hivemq-operator`](./tests-hivemq-operator).

## Verifying charts

> To sign our Helm charts, we use PGP keys when packaging our charts. These signed charts have a `.prov` file generated alongside the packaged chart, which includes the chart and the signature.

Reviewer: ```suggestion
We sign the HiveMQ Helm charts with PGP keys during the packaging. These signed charts have a `.prov` file generated alongside the packaged chart, which includes the chart and the signature.
```
